tests/e2e/tests/pilot/testdata/pilot.yaml.tmpl

```diff
@@ -42,6 +42,8 @@ spec:
   type: LoadBalancer
 {{end}}
   ports:
+  - port: 8080
```

Comment:
How much is it going to be to use the helm charts here rather than these testdata template files? (Does it make sense to do in this PR?)

Reply:
Obviously not in this PR... Mandar is working on it, and it's a very different problem.

Reply:
It almost never makes sense to ask for more features in a PR ( sometimes it makes sense to ask to split the PR ).

Reply:
(it's not trivial - the pilot init is very different and incompatible )